project_task_description_template/views/task_templates.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- pylint:disable=name-of-check-to-skip -->
```

Comment:
**question:** does this line do anything?

Reply:
Thanks, I'll remove it